index.html

```diff
@@ -0,0 +1,2409 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+    <link rel="shortcut icon" href="images/icon.jpg" />
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <!-- Bootstrap v.4.3.1 CSS -->
+    <link rel="stylesheet" href="css/bootstrap.min.css" />
+    <!-- Style CSS -->
```

Comment:
:+1: FYI It's easier to use the bootstrap cdn instead of downloading the compiled files. [Use Bootstrap](https://getbootstrap.com/docs/4.3/getting-started/download/)